src/Microsoft.AspNetCore.SignalR.Client.TS/Formatters.ts

```diff
@@ -0,0 +1,146 @@
+﻿import { Message, MessageType } from './Message';
+
+let knownTypes = {
+    "T": MessageType.Text,
+    "B": MessageType.Binary,
+    "C": MessageType.Close,
+    "E": MessageType.Error
+};
+
+function splitAt(input: string, searchString: string, position: number): [string, number] {
+    let index = input.indexOf(searchString, position);
+    if (index < 0) {
+        return [input.substr(position), input.length];
+    }
+    let left = input.substring(position, index);
+    return [left, index + searchString.length];
+}
+
+export namespace SSE {
```

Comment:
Rename to something more specific. Hard to assume what SSE is without looking at the code.